hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricDescriptorConstants.java

```diff
@@ -18,7 +18,7 @@
 
 /**
  * Class holding constants for metric names, prefixes, discriminators etc.
- * <p/>
+ * <p></p>
```

Comment:
btw, `<p>` is enough :)

Reply:
I checked in my IntelliJ doc viewer and `<p>` didn't really separate the two paragraphs, it was just like a new line and I wanted to highlight the note. Maybe it's a rendering issue.

Reply:
Could be. The 'official' Oracle [guide](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#generalform) says:

> If you have more than one paragraph in the doc comment, separate the paragraphs with a `<p>` paragraph tag, as shown.